### PR TITLE
Fix for releasing to melpa.org part2

### PR DESCRIPTION
**Pull request by masutaka**
I still have changes to release to melpa. Please review.

Related to https://github.com/melpa/melpa/pull/4828#issuecomment-313896771

### CODE REVIEW

**Comment by thierryvolpiatto** [comment]
Thanks, merged!

File: helm-elscreen.el [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+;;; helm-elscreen.el --- Elscreen with helm interface -*- lexical-binding: t -*-
 
 ;; Copyright (C) 2012 ~ 2017 Thierry Volpiatto <thierry.volpiatto@gmail.com>
 
 ;; Author: Thierry Volpiatto <thierry.volpiatto@gmail.com>
 ;; Version: 1.0
 ;; Keywords: files, convenience
 ;; URL: https://github.com/emacs-helm/helm-elscreen
-;; Package-Requires: ((emacs "24"))
+;; Package-Requires: ((emacs "24") (helm "2.8.0") (elscreen "0") cl-lib)
```
Comment (purcell) [inline helm-elscreen.el:9]: Here you should include the version for `cl-lib`, ie. `(cl-lib "0.5")`. Older versions of `package.el` can't parse the format you're using here, so it's best to always use the old format.

Comment (masutaka) [reply helm-elscreen.el:9]: Thanks. I didn't know it. I referred to https://www.gnu.org/software/emacs/manual/html_node/elisp/Library-Headers.html.

Just created #3

**Comment by thierryvolpiatto** [comment]
Steve Purcell <notifications@github.com> writes:

> Here you should include the version for cl-lib, ie. (cl-lib
> "0.5"). Older versions of package.el can't parse the format you're
> using here, so it's best to always use the old format.

Done.

Thanks Steve.

--
Thierry
Gpg Key fingerprint = 6CEC 7081 AB33 E251 4AB8  5FC2 28D1 7F53 59F2 9997

**Pull request merged**